app/views/books/show.html.erb

```diff
@@ -2,109 +2,18 @@
 
 <div class="container-fluid">
 	<div class="row">
-		<div class="col-sm-3 d-flex flex-column justify-content-center align-items-center p-2">
-			<% if (@book["covers"]) %>
-				<%= image_tag "http://covers.openlibrary.org/b/id/#{@book["covers"][0]}-L.jpg", class: "book_image" %>
-			<% else %>
-				<%= image_tag "index.png", class: "book_image" %>
-			<% end %>
+		
+		<%= react_component("BookDetails", { book: @book, preview: @preview, author: @author }) %>
 
-			<% if (@preview) %>
-				<%= link_to "Preview", "#{@preview}", class: "btn btn-secondary preview_btn" %>
-			<% else %>
-				<%= button_to "Resource not available", "#", disabled: true, class: "btn btn-secondary disabled" %>
-			<% end %>
 
-		</div>
-		<div class="col-sm-9 p-2">
-			<h1><%= @book["title"] %></h1>
-			<% if (@author == nil) %>
-				<h6>author information not available</h6>
-			<% else %>
-				<h6>by <%= render plain: @author %>  </h6>
-			<% end %>
-			<h2 class="text-secondary text-wrap">Overview</h2>
+		
+	</div>	
 
-			<div class="overview">
-				<% if (@book["description"]) %>
-					<%= @book["description"] %>
-				<% else %>
-					<%= render plain: "No Overview available" %>
-				<% end %>
-			/div>		
 
-		</div>
-	</div>
+		<%= react_component("Review", { Reviews: @reviews, currentUser: current_user, bookId: @id }) %>
```

Comment:
It's important to follow consistent naming across the project. Most of the time the convention around naming will be defined by the platform which you're using try to see what's the convention for this.

Can we make it like this ?
```suggestion
		<%= react_component("Review", { reviews: @reviews, currentUser: current_user, bookId: @id }) %>
```

Reply:
@RathanKumar actually, I already have a reviews variable inside the "Review" Component, so I have kept the props name as Reviews. But will check the correct naming convention to use here.